src/Symfony/Component/Process/Process.php

```diff
@@ -339,9 +339,11 @@ public function start(callable $callback = null, array $env = [])
         }
 
         $envPairs = [];
-        foreach ($env as $k => $v) {
-            if (false !== $v && 'argc' !== $k && 'argv' !== $k) {
-                $envPairs[] = $k.'='.$v;
+        if(\is_array($env)){
```

Comment:
The code says everywhere that `env` should be an array, here most likely you need to think about the **explicit** merging of two arrays, and not about checking for an array.